lib/java/jvm-interop/src/main/java/org/enso/jvm/interop/impl/OtherJvmLoader.java

```diff
@@ -26,14 +28,35 @@ final class OtherJvmLoader extends URLClassLoader implements TruffleObject {
             .build();
   }
 
-  void addToClassPath(String file) {
+  final void addToClassPath(String file) {
     try {
       addURL(new File(file).toURI().toURL());
     } catch (MalformedURLException ex) {
       ex.printStackTrace();
     }
   }
 
+  final void findLibraries(TruffleObject obj) {
+    this.findLibraries = obj;
+  }
+
+  @Override
+  protected String findLibrary(String libName) {
+    if (this.findLibraries != null) {
+      try {
+        var iop = InteropLibrary.getUncached();
+        var mayBePath = iop.execute(this.findLibraries, libName);
```

Comment:
With the 6af4a5a7209b11b551af97a663a45b68432f0581 we can execute:
```
sbt:enso> runEngineDistribution --vm.D=polyglot.enso.classLoading=Standard.Base:hosted,guest --run test/Image_Tests
174 tests succeeded.
0 tests failed.
0 tests skipped.
1 groups skipped.
```
because now the "dual JVM" can `OpenCV.loadShared()` without any issues.